actor/echo-httpserver-preview2/Cargo.toml

```diff
@@ -1,22 +1,18 @@
 [package]
-name = "test"
-#name = "{{project-name}}"
+name = "{{project-name}}"
 version = "0.1.0"
-authors = [ "" ]
+authors = [""]
 edition = "2021"
 
 [lib]
 crate-type = ["cdylib", "rlib"]
-#name = "{{to_snake_case project-name}}"
+name = "{{to_snake_case project-name}}"
 
 [dependencies]
 futures = "0.3"
 http = "0.2.9"
 serde = { version = "1.0.186", features = ["derive"] }
 serde_json = "1.0"
-wasmbus-rpc = "0.14"
-wasmcloud-actor = { git = "https://github.com/wasmCloud/wasmCloud.git", version = "0.1.0", features = [ "component", "rand", "uuid" ] }
-wasmcloud-interface-httpserver = "0.11"
```

Comment:
It's so weird to not see these dependencies, in a good way!

Reply:
I know, it makes me very happy